tests/utils.go

```diff
@@ -3606,7 +3606,7 @@ func CreateISCSITargetPOD(containerDiskName cd.ContainerDisk) (iscsiTargetIP str
 	image := fmt.Sprintf("%s/cdi-http-import-server:%s", flags.KubeVirtUtilityRepoPrefix, flags.KubeVirtUtilityVersionTag)
 	resources := k8sv1.ResourceRequirements{}
 	resources.Limits = make(k8sv1.ResourceList)
-	resources.Limits[k8sv1.ResourceMemory] = resource.MustParse("256M")
+	resources.Limits[k8sv1.ResourceMemory] = resource.MustParse("512M")
```

Comment:
Why we need to increase the memory requirements in the tests for Go 1.13?

Reply:
https://github.com/kubevirt/kubevirt/pull/3945#issuecomment-670357869